### PR TITLE
Adds missing tooltip to the cancel button in file transfer widget

### LINKED ISSUE #1740: Hovering over the "x"/ cancel transfer button doesn't indicate this button is to cancel a transfer. EOS 6 beta 1
## Problem
I don't know if the title is worded correctly or not (English is my second language and I do lack vocabulary at times ), nor do I know if this is a big deal to you guys or not, but I was getting ready to move over to beta 2 and was attempting to transfer my home folder onto an external storage to copy and paste it onto beta 2. Copied the whole home folder and pasted it into the external storage and the normal little window with the progress bar was showing. Hovered over the "x" circular button expecting a tiny box with "cancel" text in it, but there wasn't any. I don't know if that was intentional or not, but I am a person who like small details and wanted to bring that to your attention :)
here is a screenshot: (Note that the cursor is on the circular "x" but for some reason, the screenshot doesn't capture the cursor)

Screenshot from 2021-06-16 20-59-08

## Proposal
Hovering over a progress bar where it has an "x" at the end of to cancel an ongoing action should show a tiny box that says "cancel" or something similar to indicate that clicking this "x" would cancel the activity. I don't know, I hope that this explains this correctly. 
Thanks

### PR DESCRIPTION
**Pull request by Eldhrimer**
Fixes #1740 

PS: This is my first PR ever, so excited.

### CODE REVIEW

**Comment by Eldhrimer** [comment]
Just a note, I was thinking of writing "Cancel file transfer" in the tooltip, but I don't fully understand how translatable strings work, so instead I used the "Cancel" string that I saw it was already in used in .po files. I like the former because it's more explanatory, though

**Comment by jeremypw** [comment]
Congratulations on your first PR! First of many hopefully.  

You do not need worry too much about translatable strings except to make sure that they are easily translated into any language (using comments to translators if necessary).  You usually only need put the desired string inside the `_()` function. The rest happens automatically.

 "Cancel file transfer" is not always appropriate as the progress widget could appear during any file operation that takes longer than a couple of seconds.  As the button is closely associated with the status and details labels I am not sure that added information in the tooltip is strictly necessary.

**Comment by jeremypw** [comment]
@Eldhrimer You have a lint error that needs fixing - whitespace at the end of a line.  Make sure that your editor is set to strip trailing whitespace when saving a code file.  You can run the lint check locally before committing by installing the `io.elementary.vala-lint` package and running `io.elementary.vala-lint -d <source-root>` in a terminal.

File: src/View/Widgets/ProgressInfoWidget.vala [hunk 1]
```diff
@@ -79,6 +79,8 @@ public class Files.Progress.InfoWidget : Gtk.Grid {
         info.finished.connect (() => {
             destroy ();
         });
+        
+        button.set_tooltip_text (_("Cancel"));
```
Comment (tintou) [inline src/View/Widgets/ProgressInfoWidget.vala:83]: Prefer setting the property directly for better readability
```suggestion
        button.tooltip_text = _("Cancel");
```

**Comment by Eldhrimer** [comment]
Thanks for your responses!

@jeremypw I understand better about transalatable strings, it's better to stick to "Cancel" then. And I have enabled the plugin for stripping trailing whitespaces in Code!

@tintou I have commited with your modifications, I'll take note of this preferred style for future contributions.

Thanks!

**Review by danrabbit — CHANGES_REQUESTED** [review]
Hey thanks for making this pull request! My only suggestion would be to set this property where the button is initialized. This is part of the latest version of our code style guide which you can find here: https://docs.elementary.io/develop/writing-apps/code-style#initialize-objects-with-properites

In this case, that would mean modifying the bit at line 62 from this:

```
        var button = new Gtk.Button.from_icon_name ("process-stop-symbolic", Gtk.IconSize.BUTTON);
        button.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
```

to this
```
        var button = new Gtk.Button.from_icon_name ("process-stop-symbolic", Gtk.IconSize.BUTTON) {
            tooltip_text = _("Cancel")
        };
        button.get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
```

**Comment by Eldhrimer** [comment]
I pushed a new commit with changes requested by @danrabbit

**Review by danrabbit — APPROVED** [review]
Fantastic! Thanks for your contribution and solving this issue :tada:

**Pull request merged**